src/concrete/ml/sklearn/base.py

```diff
@@ -1311,9 +1318,12 @@ def fit(self, X: Data, y: Target, **fit_parameters):
         assert self.sklearn_model is not None, self._sklearn_model_is_not_fitted_error_message()
 
         # Convert the tree inference with Numpy operators
+        enable_rounding = os.environ.get("USE_ROUNDING", "True") == "True"
```

Comment:
worth putting a warning if it's False I would say btw !

Reply:
Added